Tasks/ANT/anttask.ts

```diff
@@ -78,14 +80,28 @@ function processAntOutputLine(line) {
 
 async function doWork() {
 
-    function enableCodeCoverage() {
+    function execEnableCodeCoverage(): Q.Promise<string> {
+        return enableCodeCoverage()
+            .then(function (resp) {
+                tl.debug("Enabled code coverage successfully");
+                return "CodeCoverage_9064e1d0";
+            }).catch(function (err) {
+                tl.warning("Failed to enable code coverage: " + err);
+                return "";
+            });
+    };
+
+    function enableCodeCoverage(): Q.Promise<any> {
```

Comment:
Per above, all the code in this function can be pushed down into the ccenabler module/class.  The only code needed in this main task file is if isCodeCoverageOpted getTool enableCodeCoverage.